README.md

```diff
@@ -11,316 +11,264 @@
+#### Deepchecks Monitoring Installation
 
-> **Recommended - download the code and run it locally** on the built-in dataset and (optional) model, or **replace them with your own**.
-
-
-### 🚀 See Our Checks Demo
-
-Play with some of the existing checks in our [Interactive Checks Demo](
-   https://checks-demo.deepchecks.com/?check=No+check+selected&utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=try_it_out), 
-and see how they work on various datasets with custom corruptions injected.
+If you're using deepchecks also for production monitoring,
+you can deploy a hobby instance in one line on Linux/MacOS (Windows is WIP!) with Docker:
 
+```
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/deepchecks/monitoring/main/deploy/deploy-oss.sh)"
+```
+This will automatically download the necessary dependencies and start the application locally.
+Installing our open source service is an excellent way for local usage, note however, 
+that it won't scale to support real-time production usage.
 
-## 📊 Usage Examples
+Check out the full installation instructions for deepchecks monitoring [here](https://docs.deepchecks.com/monitoring/stable/installation/index.html). 
 
-### Running a Suite
+</details>
 
-A [Suite](#suite) runs a collection of [Checks](#check) with optional
-[Conditions](#condition) added to them.
+### 🏃‍♀️ Quickstarts
 
-Example for running a suite on given
-[datasets](https://docs.deepchecks.com/stable/user-guide/tabular/dataset_object.html?utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=running_a_suite)
-and with a [supported
-model](https://docs.deepchecks.com/stable/user-guide/supported_models.html?utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=running_a_suite):
 
-```python
-from deepchecks.tabular.suites import model_evaluation
-suite = model_evaluation()
-result = suite.run(train_dataset=train_dataset, test_dataset=test_dataset, model=model)
-result.save_as_html() # replace this with result.show() or result.show_in_window() to see results inline or in window
-```
+<details open>
+   <summary>
+      <h4>
+         Deepchecks Testing Quickstart
+      </h4>
+   </summary>
 
-Which will result in a report that looks like this:
+Jump straight over to the respective quickstart docs:
+- [Tabular Quickstarts](https://docs.deepchecks.com/stable/tabular/auto_tutorials/quickstarts/index.html)
+- [NLP Quickstarts](https://docs.deepchecks.com/stable/nlp/auto_tutorials/quickstarts/index.html)
+- [Vision Quickstarts](https://docs.deepchecks.com/stable/vision/auto_tutorials/quickstarts/index.html) 
+to have it up and running on your data.
 
+You'll then be able to view all the chosen checks checks and inspect their status and results, 
+with the visual output that looks like this:
 
 <p align="center">
    <img src="docs/source/_static/images/general/model_evaluation_suite.gif" width="800">
```

Comment:
I know it's not new, but now that I'm looking at it, it looks low-res...

Reply:
I think I played a bit around with the gif configurations and chose that eventually, don't remember the exact considerations... though personally I think there are some advantages to it - it give the feeling of a high-level (visual) overview and it's ok that you can't really dive into the text, anyway it's not really possible. Also it looks more gif-y and less like a video which is nice.
Do you think worth redoing? (possible but if so I'll do it in a future PR after the launch. and then need to fix also the monitoring gif which is even more low-res).